File: Native/Test/Fabric/create-node.js
FABRIC = require('Fabric').createClient();

node = FABRIC.DependencyGraph.createNode("foo");
node.addMember( "foo", "Integer", 17 );
console.log( node.getData( "foo" ) );
node.setData( "foo", 42 );
console.log( node.getData( "foo" ) );

FABRIC.flush();

File: Native/Test/Python/create-node.py
#
#  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
#

import fabric
fabricClient = fabric.createClient()

node = fabricClient.DependencyGraph.createNode("foo")
node.addMember( "foo", "Integer", 17 )
print( node.getData( "foo" ) )
node.setData( "foo", 42 )
print( node.getData( "foo" ) )

fabricClient.close()
